CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing
+
+Hey, welcome to the party! 🎉
+
+Thank you so much for contributing to EasyMDE. 😘
+
+
+## Asking questions, suggesting wonderful ideas or reporting bugs
+
+You can [submit an issue️](https://github.com/Ionaru/easy-markdown-editor/issues/new) on this GitHub repository.
+
+
+## Coding
+
+### 📦 Prerequisites
+
+You need node.js and npm.
+
+To install them on Debian-based systems:
+
+```bash
+curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
+sudo apt-get install -y nodejs
+echo -e "nodejs version:\t$(nodejs -v) \nnpm version:\t$(npm -v)"
+# check that you have node.js and npm.
+```
+
+For other systems, please [read the official page](https://nodejs.org/en/download/).
+
+
+### 🏗️ Installation
+
+Here we go! 🤠 First, clone this repository:
+
+```bash
+git clone https://github.com/Ionaru/easy-markdown-editor.git
+cd easy-markdown-editor
+```
+
+Then install the EasyMDE with npm:
+
+```bash
+npm install
+```
+
+Yay! You are ready! 🍾
+
+
+### ⤴️ Creating a pull request
+
+1. First, [create a fork of this project](https://github.com/Ionaru/easy-markdown-editor/fork), and copy the https URL (*clone or download* button) of your project (something like https://github.com/YOUR_USERNAME/easy-markdown-editor.git );
+2.
+- a) If you already cloned and worked on the project: `git remote add source https://github.com/Ionaru/easy-markdown-editor.git`;
+- b) otherwise, clone your fork: `git clone https://github.com/YOUR_USERNAME/easy-markdown-editor.git`;
```

Comment:
Formatting isn't great here in the rendered markdown, I think this makes it better:

2. a) If you already cloned and worked on the project: `git remote add source https://github.com/Ionaru/easy-markdown-editor.git`;
b) otherwise, clone your fork: `git clone https://github.com/YOUR_USERNAME/easy-markdown-editor.git`;